sql/updates/latest-dev.sql

```diff
@@ -11,7 +11,7 @@ DROP FUNCTION _timescaledb_internal.partitioning_column_to_pretty;
 DROP FUNCTION _timescaledb_internal.range_value_to_pretty;
 -- end of do not reorder
 DROP VIEW IF EXISTS timescaledb_information.compressed_chunk_stats;
-DROP VIEW If EXISTS timescaledb_information.compressed_hypertable_stats;
+DROP VIEW IF EXISTS timescaledb_information.compressed_hypertable_stats;
```

Comment:
Quick question: how this update script is tested now?

Reply:
it is run during update test